recipes-extended/onl/files/delta-ag7648/0004-x86-64-delta-ag7648-i2c-mux-setting.ko-added-additio.patch

```diff
@@ -8,6 +6,7 @@ Subject: [PATCH 2/8] x86-64-delta-ag7648-i2c-mux-setting.ko: added additional
 
 Delta seems to have different settings for this. Added OEM vendor
 as well
+Upstream-Status: Inappropriate [code not upstream]
```

Comment:
This is the one instance where the commit message footer is not separated from the rest of the body by an empty line. Do you want to change that?

Reply:
I just did.